### PR TITLE
Fix/aut 2806/low risk fixes

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**

### CODE REVIEW

File: views/package-lock.json [hunk 1]
```diff
@@ -2657,9 +2657,9 @@
                     "dev": true
                 },
                 "socket.io-parser": {
-                    "version": "3.3.2",
```
Comment (andreluizmachado) [inline views/package-lock.json:2660]: Are we sure that it's a good idea to change the package-lock directly?
is It a good practice?
what if we update the root dependency?
Screenshot 2023-01-25 at 10 14 54

**Review by bziondik — APPROVED** [review]
LGTM

File: views/package-lock.json (hunk 1 above)
Comment (bartlomiejmarszal) [reply views/package-lock.json:2660]: I did not change that in .lock file. It was done along with npm update process.

**Review by pnal — APPROVED** [review]

**Review by andreluizmachado — APPROVED** [review]
LGTM

**Comment by Karol-Stelmaczonek** [comment]
Closing, those changes are already in current develop.

**Pull request closed without merging**